### ISSUE TITLE
Add Formie-specific attribute to Form Alerts

### ISSUE THREAD

**Issue by chrismlusk** [verbb/formie#1509]
### What are you trying to do?

I have a form with a fair bit of customization added. One of my features involves injecting a notification below a certain field when a set of conditions are not met. (FWIW, this does not use field conditions because of the complexity involved.)

To make these alerts accessible to screen readers, I am include the `alert` role on these notifications. However, if a user then attempts an invalid submission and Formie needs to show a form alert, my custom notification interferes with the Form Theme's `showFormAlert` method, which is querying for an existing element with the `alert` role:

```js
let $alert = this.$form.parentNode.querySelector('[role="alert"]');
```

### What's your proposed solution?

Giving the Formie form alert element a unique attribute — e.g., `data-fui-form-alert` — and querying for that attribute would help avoid conflicts like this.

**Comment by engram-design** [comment]
I should mention the default alert already contains `role="alert"`, so you shouldn't need to do it yourself?

Updated for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.33"`.

**Comment by chrismlusk** [comment]
Thanks!

To clarify, I'm inserting my own alert within the form, and this happens before Formie attempts to show its alert. So when `showFormAlert` executes and queries for an element with `[role="alert"]`, it's finding my alert and inserting the form error message there instead of creating a new alert as expected.

**Comment by engram-design** [comment]
Fixed in [2.0.34](https://github.com/verbb/formie/releases/tag/2.0.34)
